### ISSUE TITLE
cannot start the watch process after initializing the project

### ISSUE THREAD

**Issue by t-cool** [dutiyesh/chrome-extension-cli#5]
Hi,

I cannot start the watch process after initializing the project.
The error is as follows:

```
$ node
Welcome to Node.js v12.17.0.
Type ".help" for more information.

$ chrome-extension-cli my-extension
Creating a new Chrome extension in /Users/username/Desktop/my-extension

Installing packages. This might take a couple of minutes.
Installing webpack, webpack-cli and few more...

npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> fsevents@1.2.13 install /Users/username/Desktop/my-extension/node_modules/watchpack-chokidar2/node_modules/fsevents
> node install.js

  SOLINK_MODULE(target) Release/.node
  CXX(target) Release/obj.target/fse/fsevents.o
  SOLINK_MODULE(target) Release/fse.node
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN notsup Unsupported engine for watchpack-chokidar2@2.0.0: wanted: {"node":"<8.10.0"} (current: {"node":"12.17.0","npm":"6.14.4"})
npm WARN notsup Not compatible with your version of node/npm: watchpack-chokidar2@2.0.0

+ copy-webpack-plugin@6.0.1
+ size-plugin@2.0.1
+ mini-css-extract-plugin@0.9.0
+ webpack-merge@4.2.2
+ webpack-cli@3.3.11
+ css-loader@3.5.3
+ webpack@4.43.0
+ file-loader@6.0.0
added 547 packages from 259 contributors and audited 547 packages in 14.956s

30 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

Generated a README file.

Initialized a git repository.

Success! Created my-extension at /Users/tcool/Desktop/my-extension
Inside that directory, you can run below commands:

  npm run watch
    Listens for files changes and rebuilds automatically.

  npm run build
    Bundles the app into static files for Chrome store.

We suggest that you begin by typing:

  1. cd my-extension
  2. Run npm run watch
  3. Open chrome://extensions
  4. Check the Developer mode checkbox
  5. Click on the Load unpacked extension button
  6. Select the folder my-extension/build

$ npm run watch

> my-extension@0.1.0 watch /Users/username/Desktop/my-extension
> webpack --mode=development --watch --config config/webpack.config.js

Invalid options object. Copy Plugin has been initialized using an options object that does not match the API schema.
 - options[0] misses the property 'patterns'. Should be:
   [non-empty string | object { from, to?, context?, globOptions?, toType?, force?, flatten?, transform?, cacheTransform?, transformPath?, noErrorOnMissing? }, ...] (should not have fewer than 1 item)
ValidationError: Invalid options object. Copy Plugin has been initialized using an options object that does not match the API schema.
    at validate (/Users/username/Desktop/my-extension/node_modules/schema-utils/dist/validate.js:88:11)
    at new CopyPlugin (/Users/username/Desktop/my-extension/node_modules/copy-webpack-plugin/dist/index.js:24:30)
    at Object.<anonymous> (/Users/username/Desktop/my-extension/config/webpack.common.js:52:5)
    at Module._compile (/Users/username/Desktop/my-extension/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (/Users/username/Desktop/my-extension/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/Users/username/Desktop/my-extension/config/webpack.config.js:5:16)
    at Module._compile (/Users/tcool/Desktop/my-extension/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (/Users/username/Desktop/my-extension/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! my-extension@0.1.0 watch: `webpack --mode=development --watch --config config/webpack.config.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the my-extension@0.1.0 watch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
```

**Comment by krapnikkk** [comment]
webpack.common.js
`new CopyWebpackPlugin(
      {
        patterns:
          [
            {
              from: '**/*',
              context: 'public',
            },
          ]
      })`

**Comment by elron** [comment]
Still not fixed.
Anyone knows the problem?

**Comment by dutiyesh** [comment]
@krapnikkk is right, the problem is related to `CopyWebpackPlugin`, its syntax of passing `"patterns"` has changed in its latest version.
I will fix it in few days and release it.

**Comment by dutiyesh** [comment]
Issue is fixed in [v0.2.1](https://github.com/dutiyesh/chrome-extension-cli/releases/tag/v0.2.1) release.

**Comment by elron** [comment]
It works, thanks!
Wonderful CLI.
